samtranslator/swagger/swagger.py

```diff
@@ -554,10 +555,15 @@ def is_valid(data):
         :param dict data: Data to be validated
         :return: True, if data is a Swagger
         """
-        return bool(data) and \
-            isinstance(data, dict) and \
-            bool(data.get("swagger")) and \
-            isinstance(data.get('paths'), dict)
+        openapi_version_supported_regex = r"\A3(\.\d)(\.\d)?$"
```

Comment:
Could you consolidate these regexes? It would be easier to maintain them in case we need to update the version in the future. I've seen two different ones, but one of them is used many more times than the other.

Reply:
Where would the right place to consolidate be?